src/lib.rs

```diff
@@ -3,19 +3,19 @@ Compiletime string literal obfuscation.
 !*/
 
 #![no_std]
-#![feature(fixed_size_array)]
 
 // Reexport these because reasons...
 #[doc(hidden)]
 pub use obfstr_impl::*;
+pub use obfstr_impl::wide;
```

Comment:
Why export those publicly? I think it's better to keep them as private impl details.

Reply:
Let's discuss in the other thread.